consensus/src/consensus.rs

```diff
@@ -28,18 +30,22 @@ pub struct ConsensusClient<R: ConsensusRpc, DB: Database> {
     phantom: PhantomData<R>,
 }
 
-#[derive(Debug)]
-pub struct Inner<R: ConsensusRpc> {
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct ConsensusStateManager<R: ConsensusRpc> {
```

Comment:
I've changed the name of this struct because `Inner` would mean nothing when outside of `ConsensusClient` context.

Reply:
💯 
Please doc it.